src/main/java/appeng/block/misc/BlockSkyCompass.java

```diff
@@ -164,4 +186,31 @@ public void addCollidingBlockToList( final World w, final BlockPos pos, final Ax
 	{
 
 	}
+
+	@Override
+	public EnumBlockRenderType getRenderType( IBlockState state )
+	{
+		return EnumBlockRenderType.ENTITYBLOCK_ANIMATED;
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public TileEntitySpecialRenderer<? extends AEBaseTile> getTESR()
+	{
+		return new SkyCompassTESR();
+	}
+
+	@Override
+	@SideOnly( Side.CLIENT )
+	public ItemMeshDefinition getItemMeshDefinition()
+	{
+		return is -> new ModelResourceLocation( "appliedenergistics2:tile.BlockSkyCompass", "normal" );
```

Comment:
So should we go for it? Java 8 can simplify a lot of things for us. And i don't think many userscan't use java 8?

Should we ask community???